### ISSUE TITLE
Warn user when AAR explosion is disabled and the internal build system is selected

### ISSUE THREAD

**Issue by stewartmiles** [googlesamples/unity-jar-resolver#100]
When AAR explosion is disabled and Unity's internal build system is selected the build will fail in various ways:

- AARs that contain multiple architectures are not stripped which can lead to load library failing on x86 or x86_64 devices.
- AARs that do not contain classes.jar result in a cryptic "Moving File Failed error" when processing the AARs

It would be nice to just warn the user about problematic AARs when explosion is disabled rather than leaving them to scratch their heads wondering what's going on.

**Comment by num0n1** [comment]
spent several hours on this, finnally find someone had the same problem

**Comment by stewartmiles** [comment]
@num0n1 I just took a look at this and even though I filed this issue I haven't been able to reproduce this failing behavior with auto-resolution turned on.

Here's what I'm doing:
- Switch build system to `Gradle`
- Wait for auto-resolve, a load of AARs are copied into the `Assets/Plugins/Android` directory
- Turn off Explode AARs setting.
- Switch build system to `Internal`
- Wait for auto-resolve, a load of AARs are copied and *exploded* into the `Assets/Plugins/Android` directory.

With this behavior, builds work as expected all broken -license AARs (thanks Google Play services folks) are fixed by the explode step.

So I'm wondering what's different in your case?
- Which Unity version?
- Which steps did you go through that got you into a broken state?
